src/test/pybind/test_rbd.py

```diff
@@ -762,6 +762,14 @@ def test_remove_snap(self):
         self.image.remove_snap('snap1')
         eq([], list(self.image.list_snaps()))
 
+    def test_remove_snap_ImageNotFound(self):
+        eq([], list(self.image.list_snaps()))
```

Comment:
Nit: no need to retest listing/creating/removing snapshots -- just test remove against a non-existent snap

Reply:
thank you vevry much 
i have updated